packages/frameworks/src/frameworks.ts

```diff
@@ -875,6 +875,43 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'SvelteKit',
+    slug: 'sveltekit',
+    demo: 'https://sveltekit.examples.vercel.com',
+    logo: 'https://raw.githubusercontent.com/vercel/vercel/main/packages/frameworks/logos/svelte.svg',
+    tagline:
+      'SvelteKit is a framework for building web applications of all sizes.',
+    description:
+      'A SvelteKit app optimized to work for serverless.',
+    website: 'https://kit.svelte.dev',
+    detectors: {
+      every: [
+        {
+          path: 'package.json',
+          matchContent:
+            '"(dev)?(d|D)ependencies":\\s*{[^}]*"@sveltejs/kit":\\s*".+?"[^}]*}',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install` or `npm install`',
+      },
+      buildCommand: {
+        placeholder: '`npm run build` or `svelte-kit build`',
+      },
+      devCommand: {
+        placeholder: 'svelte-kit dev',
+      },
+      outputDirectory: {
+        placeholder: 'public',
+      },
+    },
+    devCommand: 'svelte-kit dev',
```

Comment:
Does it accept the `$PORT` env var already? If not then we need to do something like `svelte-kit dev --port $PORT`

Reply:
I can't find anything in the docs about this, this is all I found: https://github.com/sveltejs/kit/blob/073fc3b5ff7500087b9ae5da8f635430b4d96267/packages/kit/src/cli.js#L55

Reply:
Let's pass `--port` here and in the placeholder too

https://github.com/sveltejs/kit/blob/073fc3b5ff7500087b9ae5da8f635430b4d96267/packages/kit/src/cli.js#L75

Reply:
You can also verify this by running `vc dev` and modify your projects Dev Command to see if it works